src/python/grpcio/grpc/beta/_server_adaptations.py

```diff
@@ -170,7 +170,10 @@ def stop_request_pipe(timeout):  # pylint: disable=unused-argument
         thread_joined.set()
 
     request_pipe_thread = _common.CleanupThread(
-        stop_request_pipe, target=pipe_requests)
+        stop_request_pipe,
+        target=pipe_requests,
+        name='Thread-gRPC-StopRequestPipe',
```

Comment:
Ditto here (I didn't realize the cleanup was a secondary part), this should probably be called `Thread-gRPC-PipeRequests`?

Reply:
`Thread-gRPC-pipe_requests` if we want to be consistent down to letter case.